pkgs/build-support/node/fetch-npm-deps/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, stdenv, rustPlatform, Security }:
+
+rec {
+  fetch-npm-deps = rustPlatform.buildRustPackage {
```

Comment:
I think `fetch-npm-deps` and `fetchNpmDeps` being named pretty much the same is a bit confusing, maybe we could rename this to `generateNpmDeps`?

Please also set `meta` and include yourself as a maintainer, together with a description.